src/librbd/mirror/snapshot/PromoteRequest.cc

```diff
@@ -35,16 +35,22 @@ using librbd::util::create_rados_callback;
 template <typename I>
 void PromoteRequest<I>::send() {
   CephContext *cct = m_image_ctx->cct;
+  ldout(cct, 15) << "m_rollback_snap_id: " << m_rollback_snap_id << dendl;
```

Comment:
This would be better added to the constructor

Reply:
Moved it in to the block.